src/styles/polygons/polygons_fragment.glsl

```diff
@@ -35,9 +40,14 @@ void main (void) {
     vec4 color = v_color;
     vec3 normal = TANGRAM_NORMAL;
 
-    // Normal material texture (fragment lighting only)
-    #if defined(TANGRAM_LIGHTING_FRAGMENT) && defined(TANGRAM_MATERIAL_NORMAL_TEXTURE)
-        calculateNormal(normal);
```

Comment:
Yep - just checked
http://tangrams.github.io/tangram-docs/?material/ex-normals.yaml#19/40.70533/-74.00975
locally using the 0.7 release, and the normal map isn't being applied.

On Thu, Apr 14, 2016 at 2:03 PM, Brett Camper <notifications@github.com>
wrote:

> In src/styles/polygons/polygons_fragment.glsl
> <https://github.com/tangrams/tangram/pull/282#discussion_r59763290>:
>
> > @@ -35,9 +40,14 @@ void main (void) {
> >      vec4 color = v_color;
> >      vec3 normal = TANGRAM_NORMAL;
> >
> > -    // Normal material texture (fragment lighting only)
> > -    #if defined(TANGRAM_LIGHTING_FRAGMENT) && defined(TANGRAM_MATERIAL_NORMAL_TEXTURE)
> > -        calculateNormal(normal);
>
> Ack, that is a mistake if we lost regular material normal maps! Let me
> look through history... but yes I think that allowing them to be compound
> makes sense.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/tangram/pull/282/files/0bcb36919e2e553c47a179fe5c07bcd9e27e4491#r59763290>
>